components/automate-chef-io/content/docs/on-prem-builder.md

```diff
@@ -93,6 +93,70 @@ products on the command line. For example:
 
 Accept the license with `y`.
 
+To deploy Chef Habitat Builder with the Chef Automate UI and Auth services only, specify the `builder`
```

Comment:
[suggestion] It'd be good to let the user know up-front that installing this way is an option. What comes to my mind is opening the "Deploy Chef Automate ..." section with something like "You can deploy Chef Habitat Builder either with a full Chef Automate installation or with the Chef Automate Auth stack only," then giving each type of deploy its own section header.